lanweatherd/test_scripts/req_test.py

```diff
@@ -0,0 +1,15 @@
+import zmq
+
+ctx = zmq.Context()
+
+# pylint: disable=E1101
+socket = ctx.socket(zmq.REQ)
+
+socket.connect("tcp://localhost:5680")
+
+socket.send(b"launch the nukes")
```

Comment:
What’s the b that you put before the send message?

Reply:
[python bytes literal](https://docs.python.org/3/library/stdtypes.html#bytes-objects)